tests/calculation.test.js

```diff
@@ -0,0 +1,61 @@
+const describe = require('mocha').describe
+const it = require('mocha').it
+const expect = require('chai').expect
+const strictequals = require('../calculation')
+
+describe('calculation', () => {
+  describe('strictequals', () => {
+    it('returns true if the values are the same and does not type coerce', () => {
+      //create test data 
+      const num1 = 1
+      const num2 = '1'
+      //call the function
+      const checkvalue = strictequals(num1, num2)
+      //make assertions 
+      expect(checkvalue).to.equal(false)
+    })
+  })
+})
+
+describe('calculation', () => {
+  describe('strictequals', () => {
```

Comment:
Describes are for high level formatting purposes. You already have describes above that tell me it's the calculation file and strictequals function we are testing. No need to repeat that for each test. You can put as many `it()` calls inside a describe as you want.